docs/config_yaml.md

```diff
@@ -2,60 +2,121 @@
 
 The GNSS-Acoustic positioning performed by GNATSS is governed by a configuration
 file named "config.yaml". This is a text file with a YAML format that defines
```

Comment:
This file name is now not a requirement. So probably just say something to the effect of just a YAML configuration file.

Reply:
Done.